disciplina.cabal

```diff
@@ -62,6 +62,7 @@ library
                      , Disciplina.Educator.Block
                      , Disciplina.Educator.Txs
                      , Disciplina.Educator.State
+                     , Disciplina.Educator.SizedMerkleTree
```

Comment:
I think that `SizedMerkleTree` module fits better in `Disciplina.Crypto` package. It isn't very specific to `Educator` logic, it's a generic structure, even though right now it's only used by `Educator` node.

Reply:
Ok. I will move it to Disciplina.Crypto